jetstream/validation.py

```diff
@@ -0,0 +1,84 @@
+"""Validation helpers."""
+
+import os
+import toml
+from datetime import datetime
+from pathlib import Path
+
+from .analysis import Analysis
+from .config import AnalysisConfiguration, AnalysisSpec, OutcomeSpec, PLATFORM_CONFIGS
+from .dryrun import DryRunFailedError
+from .experimenter import Experiment, ExperimentCollection
+from .external_config import OUTCOMES_DIR
+
+
+def validate_config(path: str) -> bool:
+    """Validate all config files in the provided path."""
+    config_files = [p for p in path if os.path.isfile(p)]
+
+    collection = ExperimentCollection.from_experimenter()
+
+    for file in config_files:
+        print(f"Validate {file}")
+
+        if Path(file).parent.parent.name == OUTCOMES_DIR:
+            # file is an outcome snippet
+            outcomes_spec = OutcomeSpec.from_dict(toml.load(file))
+            platform = Path(file).parent.name
+
+            if platform not in PLATFORM_CONFIGS:
+                print(f"Platform '{platform}' is unsupported.")
+                return False
+            else:
+                for app_id in PLATFORM_CONFIGS[platform].app_ids:
```

Comment:
Does it make sense to dry run outcome configs for every app_id? Or should we just select one?

Reply:
It seems like we should be able to choose either nightly or release; nightly is the most permissive since probes should land there sooner, which seems like the right move to me.